examples/vtkplotter.ipynb

```diff
@@ -0,0 +1,141 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install dependencies for this example\n",
+    "# Note: This does not include itkwidgets, itself\n",
+    "import sys\n",
+    "!{sys.executable} -m pip install vtkplotter"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(Actor)0x7fa08587bca8"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from vtkplotter import *\n",
```

Comment:
Avoiding `import *`, a best practice, will keep the global namespace clear and help people learning vtkplotter understand where functions come from.